pagination/keysetpagination/paginator.go

```diff
@@ -61,12 +87,28 @@ func (p *Paginator) ToOptions() []Option {
 //	q := c.Where("foo = ?", foo).Scope(keysetpagination.Paginate[Item](paginator))
 func Paginate[I Item](p *Paginator) pop.ScopeFunc {
 	var item I
-	id := (&pop.Model{Value: item}).IDField()
+	model := &pop.Model{Value: item}
+	id := model.IDField()
 	return func(q *pop.Query) *pop.Query {
 		eid := q.Connection.Dialect.Quote(id)
+
+		tokenParts := parseToken(id, p.Token())
+		idValue := tokenParts[id]
+		if column, ok := model.Columns().Cols[p.additionalColumn.name]; ok {
+			quoteName := q.Connection.Dialect.Quote(column.Name)
```

Comment:
From a quick look at the code in pop, `(Dialect).Quote` does not prevent SQL injection. We should clarify in the documentation for the function that no untrusted input should be passed down as the column name (or direction).
Do you have other ideas how we can prevent injection here? We could e.g. use reflection to verify that the string is a valid `db` struct tag, although that would be problematic with joins where you have to use dot notation to specify the table. But we would have the same problem with the ID field not being scoped in SQL.

Reply:
The line above this prevents SQL injection, as it checks whether, the supplied field is part of the model. If it's not, we ignore it and go to the `else` clause. IMO that's enough, but feel free to exploit ;)

Reply:
Ah ok, that's great then.